index.js

```diff
@@ -64,19 +64,51 @@ menuLinkContactElement.addEventListener('click', () => {
 });
 
 // ----------------------- contact validation ---------------------------------
-const formElement = document.getElementById('contact-form');
+// const formElement = document.getElementById('contact-form');
 // const formNameElement = document.getElementById('name');
-const formEmailElement = document.getElementById('email');
+// const formEmailElement = document.getElementById('email');
 
```

Comment:
[OPTIONAL]Please it is important to follow microverse clean code policy. Please kindly remove unused code for it makes your code base look untidy and it may also lead to you experiencing errors.